specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/examples/StorageAccountCreateObjectReplicationRestorePointMarker.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is there any relationship with this feature and "objectReplicationPolicies" ?
https://learn.microsoft.com/en-us/rest/api/storagerp/object-replication-policies

Reply:
reverted this change from PR it is not in scope for this release.